editors/code/src/run.ts

```diff
@@ -96,6 +96,30 @@ export class RunnableQuickPick implements vscode.QuickPickItem {
     }
 }
 
+export function prepareEnv(runnable: ra.Runnable, runnableEnvCfg: RunnableEnvCfg): Record<string, string> {
+    const env: Record<string, string> = { "RUST_BACKTRACE": "short" };
+
+    if (runnable.args.expectTest) {
+        env["UPDATE_EXPECT"] = "1";
+    }
+
+    Object.assign(env, process.env as { [key: string]: string });
```

Comment:
For me, this makes no sense, as a new process will get a copy automatically. But as this code exists from the very beginning I left it as is in case I've missed something.

Reply:
Apparenty, if you don't copy `process.env`, you won't play nicely with extensions that modify env, like https://marketplace.visualstudio.com/items?itemName=arrterian.nix-env-selector.

Though, I haven't looked into this closely (was contributed in the PR), it might make sense to look at git blame.

Reply:
Thanks, I suspected something like that :) Then let it stay, not a big deal anyway.